ambry-store/src/main/java/com/github/ambry/store/StoreMetrics.java

```diff
@@ -277,17 +277,9 @@ void initializeIndexGauges(String storeId, final PersistentIndex index, final lo
     }
     if (enableStoreIndexDirectMemoryUsageMetric) {
       indexes.putIfAbsent(storeId, index);
-      Gauge<Long> indexDirectMemoryUsage = () -> {
-        long start = System.nanoTime();
-        long usage = indexes.values().stream().mapToLong(PersistentIndex::getDirectMemoryUsage).sum();
-        if (logger.isTraceEnabled()) {
-          logger.trace("Time to get direct memory usage from persistent index is: {} nano seconds",
-              System.nanoTime() - start);
-        }
-        return usage;
-      };
-      registry.gauge(MetricRegistry.name(BlobStore.class, prefix + "IndexDirectMemoryUsage"),
-          () -> indexDirectMemoryUsage);
+      Gauge<Long> indexDirectMemoryUsage =
+          () -> indexes.values().stream().mapToLong(PersistentIndex::getDirectMemoryUsage).sum();
+      registry.gauge(MetricRegistry.name(BlobStore.class, "IndexDirectMemoryUsage"), () -> indexDirectMemoryUsage);
```

Comment:
QQ: Does that mean for all blobstores, they will have same IndexDirectMemoryUsage? What if some blobStore has higher usage?